gnu.elf/plugin.xml

```diff
@@ -1450,6 +1547,30 @@
                   name="%linker.option.nano"
                   valueType="boolean">
             </option>
+            <option
+                  category="arc.gnu.elf.c.linker.category.general"
+                  id="arc.gnu.elf.c.linker.option.specshs6x"
+                  name="%linker.option.specshs6x"
+                  valueType="enumerated">
+               <enumeratedOptionValue
+                     command="--specs=qemu.specs"
+                     id="arc.gnu.elf.c.linker.option.specshs6x.qemu"
+                     isDefault="true"
+                     name="%linker.option.specshs6x.qemu">
+               </enumeratedOptionValue>
+               <enumeratedOptionValue
+                     command="--specs=hl.specs"
+                     id="arc.gnu.elf.c.linker.option.specshs6x.hl"
+                     name="%linker.option.specshs6x.hl">
+               </enumeratedOptionValue>
```

Comment:
This option need an empty option value, if selected the option will not be passed at all to the compiler (for example like it is done for regular --specs option).